src/api/crypto.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { ipcMain } from 'electron';
+
+import { IPC_CHANNELS } from '@config';
+import { CryptoRequestType } from '@types';
+
+import { handleRequest, runService } from './crypto';
+
+jest.mock('@ethersproject/wallet', () => ({
+  Wallet: jest.fn().mockImplementation(() => ({
+    address: '0x4bbeEB066eD09B7AEd07bF39EEe0460DFa261520',
+    signTransaction: jest.fn().mockImplementation(() => Promise.resolve('signed'))
```

Comment:
Can we return an actually signed transaction here? That may prevent situations in the future where some library depends on a proper signed transaction hexadecimal string, but gets "signed" instead, something that broke a bunch of stuff in legacy.